api/datasets.go

```diff
@@ -153,8 +153,8 @@ func (h *DatasetHandlers) RenameHandler(w http.ResponseWriter, r *http.Request)
 	}
 }
 
-// DataHandler gets a dataset's data
-func (h *DatasetHandlers) DataHandler(w http.ResponseWriter, r *http.Request) {
+// BodyHandler gets a dataset's body
```

Comment:
I prefer the phrasing used elsewhere in the file:
// BodyHandler gets the contents of a dataset

or

// BodyHandler gets the contents of a dataset's body

It's less tautological sounding.